lib/Channel.js

```diff
@@ -219,11 +219,6 @@ class Channel {
             this.initialize()
         }
         return TwilioIPMessagingMessages.getLastMessages(this.sid, count)
-        .then((messages) => {
```

Comment:
I think this might be needed? I think the native code just returns the raw data and it's initialized into the Message class here.